frontend/ios/Runner/Info.plist

```diff
@@ -28,6 +28,8 @@
 	<string>Einlesen von QR-Codes und Abfotografieren von Dokumenten für das Bewerbungsformular.</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
 	<string>Standort auf der Karte und Akzeptanzstellen in der Umgebung anzeigen.</string>
+	<key>NSLocationAlwaysUsageDescription</key>
+	<string>Standort auf der Karte und Akzeptanzstellen in der Umgebung anzeigen.</string>
```

Comment:
Do we still need `NSLocationWhenInUsageDescription` then?

Reply:
Yes, they are shown at different spots. The thing is because we use flutter, code for both types of usage in Swift/ObjectiveC is included in the app. That means apple thinks that both dialogs could be shown even though this is wrong.

So this is just disabling a warning by apple.